pkgs/development/python-modules/uvicorn/default.nix

```diff
@@ -29,11 +33,18 @@ buildPythonPackage rec {
     wsproto
   ];
 
-  checkPhase = ''
-    $out/bin/uvicorn --help
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "h11==0.8.*" "h11"
   '';
 
-  patches = [ ./setup.patch ];
+  checkInputs = [ pytest requests ];
+  checkPhase = ''
+    pytest
+  '';
+  #checkPhase = ''
+  #  $out/bin/uvicorn --help
+  #'';
```

Comment:
With actual tests added, the old `checkPhase` should be removed.

Reply:
Yea, looks like i messed up a git rebase -i :(

Reply:
removed